### ISSUE TITLE
Could you share your training log?

### ISSUE THREAD

**Issue by polor1010** [chenjun2hao/DDRNet.pytorch#2]
Could you share your training log? 
I used the ddrnet23_slim's default setting (experiments/cityscapes/ddrnet23_slim.yaml) with 1000 Epochs but got only 0.6223 mIoU. 
thanks.

=> saving checkpoint to output/cityscapes/ddrnet23_slimcheckpoint.pth.tar
Loss: 0.396, MeanIU:  0.6223, Best_mIoU:  0.6223
[0.96390848 0.76018533 0.88509226 0.32266355 0.43711438 0.54837026
 0.62688693 0.70588259 0.91061873 0.5774142  0.93025264 0.7288157
 0.41738881 0.89958299 0.27927647 0.50435703 0.23100624 0.38669031
 0.70799136]
Epoch: [999/1000] Iter:[0/92], Time: 4.41, lr: [1.995262314968881e-05], Loss: 0.317782, Acc:0.815917
Epoch: [999/1000] Iter:[10/92], Time: 1.06, lr: [1.798967935132047e-05], Loss: 0.330938, Acc:0.799646
Epoch: [999/1000] Iter:[20/92], Time: 0.91, lr: [1.6002587191915764e-05], Loss: 0.326241, Acc:0.789909
Epoch: [999/1000] Iter:[30/92], Time: 0.85, lr: [1.3987608745552127e-05], Loss: 0.331387, Acc:0.794377
Epoch: [999/1000] Iter:[40/92], Time: 0.80, lr: [1.1939715118663247e-05], Loss: 0.329101, Acc:0.795018
Epoch: [999/1000] Iter:[50/92], Time: 0.77, lr: [9.851793876099303e-06], Loss: 0.327667, Acc:0.797766
Epoch: [999/1000] Iter:[60/92], Time: 0.76, lr: [7.71304625589395e-06], Loss: 0.324819, Acc:0.797479
Epoch: [999/1000] Iter:[70/92], Time: 0.75, lr: [5.505178031253932e-06], Loss: 0.327130, Acc:0.796518
Epoch: [999/1000] Iter:[80/92], Time: 0.74, lr: [3.190465676883869e-06], Loss: 0.324793, Acc:0.797461
Epoch: [999/1000] Iter:[90/92], Time: 0.73, lr: [6.36089096925948e-07], Loss: 0.324365, Acc:0.794952

**Comment by chenjun2hao** [comment]
@polor1010 ， this my val results:
```
2021-02-26 05:14:58,778 0 [0.978855   0.83344005 0.91876829 0.49381769 0.59358367 0.62396322
 0.67242607 0.75924213 0.9225001  0.64516499 0.9436861  0.79758483
 0.60381455 0.94177735 0.74483487 0.87634673 0.78023146 0.57579584
 0.74011868] 0.7603132443386243
2021-02-26 05:14:58,778 1 [0.96282164 0.7511382  0.88272034 0.31328483 0.42899454 0.53256223
 0.59814297 0.69861727 0.90901409 0.55433174 0.92296966 0.72051694
 0.35943582 0.89623519 0.26123333 0.46183021 0.25422931 0.34891968
 0.69806578] 0.6081612506186097
```

the ddrnet has two outputs, maybe you have the worse for test。
